src/pages/index.js

```diff
@@ -33,10 +46,58 @@ constants.profileEditButton.addEventListener("click", () => {
   profileFormValidator.resetValidation();
 });
 
-function handleProfileFormSubmit({ name, description }) {
-  user.setUserInfo({ name, description });
+function handleProfileFormSubmit({ name, about }) {
+  profileEditModal.setLoading(true);
+  api
+    .updateUserProfile({ name, about })
+    .then(() => {
+      user.setUserInfo({ name, about });
+    })
+    .then(() => {
```

Comment:
Needs correction: here you should use `finally` as you should stop the loading in any case

Reply:
Hello, thank you for a very constructive review. I've managed to fix all the issues highlighted by you, but I'm having 2 issues right now. First, if I use prepend instead of append in the Section.addItem, the card only gets prepended when initially added, on reload the latest card added gets moved all the way to the bottom, thats why I was using append because that way on reload it comes out on top. Second, I'm not sure why, but the card that has been just submitted, cannot be deleted, I get error 400, the card is not being found, but it does get deleted normally after page reload. I'm not sure if it's supposed to be happening, just wanted to clarify with you.